packages/eslint-plugin-mdx/src/rules/helpers.ts

```diff
@@ -90,9 +91,10 @@ export const getRemarkProcessor = (searchFrom: string, isMdx: boolean) => {
 
   return plugins
     .reduce((processor, pluginWithSettings) => {
-      const [plugin, ...pluginSettings] = Array.isArray(pluginWithSettings)
-        ? pluginWithSettings
-        : [pluginWithSettings]
+      const [plugin, ...pluginSettings] = arrayify(pluginWithSettings) as [
+        RemarkPlugin,
+        ...unknown[]
+      ]
```

Comment:
Does this change make sense?
It goes from a single condition, which does not require a case, and is three lines long.
To a multiple condition, which requires a cast, and is four lines long. :thinking:

Reply:
typings will be dropped on compiling, so the final codes would be simpler.